examples/DGmethods/ex_001_dycoms.jl

```diff
@@ -115,14 +119,244 @@ function Initialise_DYCOMS!(state::Vars, aux::Vars, (x,y,z), t)
+	qt_standard_dev = (qt_flucttot / (size(fluctQ, 1) * size(fluctQ, 3) * nranks) )^(0.5)
+  
+  #Variance
+	Global_Variance_rho = rho_standard_dev^2
+	Global_Variance_U = U_standard_dev^2
+	Global_Variance_V = V_standard_dev^2
+	Global_Variance_W = W_standard_dev^2
+	Global_Variance_e = e_standard_dev^2
+	Global_Variance_qt = qt_standard_dev^2
+  end
+
+ @info "ρ standard_deviation = $(rho_standard_dev)" 
+ @info "ρ Variance = $(Global_Variance_rho)"
+ @info "U standard_deviation = $(U_standard_dev)"
+ @info "U Variance = $(Global_Variance_U)"
+ @info "V standard_deviation = $(V_standard_dev)"
+ @info "V Variance = $(Global_Variance_V)"
+ @info "W standard_deviation = $(W_standard_dev)"
+ @info "W Variance = $(Global_Variance_W)"
+ @info "e standard_deviation = $(e_standard_dev)"
+ @info "e Variance = $(Global_Variance_e)"
+ @info "qt standard_deviation = $(qt_standard_dev)"
+ @info "qt Variance = $(Global_Variance_qt)"
 
+ S = zeros(Nqk, nvertelems,6)
+for eh in 1:nhorzelems
+  for ev in 1:nvertelems
+    e = ev + (eh - 1) * nvertelems
+
+    for k in 1:Nqk
+      for j in 1:Nq
+        for i in 1:Nq
+          ijk = i + Nq * ((j-1) + Nq * (k-1)) 
+          S[k,ev,1] += fluctQ[ijk, 4, e]*fluctT[ijk,5,e]
+	  S[k,ev,2] += fluctQ[ijk, 4, e]*fluctT[ijk,3,e]
+	  S[k,ev,3] += fluctQ[ijk, 4, e]*fluctQ[ijk,2,e]
+	  S[k,ev,4] += fluctQ[ijk, 4, e]*fluctQ[ijk,3,e]
+	  S[k,ev,5] += fluctQ[ijk, 4, e]*fluctQ[ijk,4,e]
+	  S[k,ev,6] += fluctQ[ijk, 4, e]*fluctQ[ijk,1,e]
+        end
+      end
+    end
+  end
+end
+S_avg=zeros(Nqk,nvertelems,6)
+for s in 1:6
+ for ev in 1:nvertelems
+   for k in 1:Nqk
+     S_avg[k,ev,s]=MPI.Reduce(S[k,ev,s], +, 0, MPI.COMM_WORLD)
+     if mpirank == 0
+       S_avg[k,ev,s] = S_avg[k,ev,s]/(Nq*Nq*nhorzelems*nranks)
+     end
+   end
+ end
+end
+OutputHF = zeros(nvertelems*Nqk)
+OutputWQVAP = zeros(nvertelems*Nqk)
+OutputWU = zeros(nvertelems*Nqk)
+OutputWV = zeros(nvertelems*Nqk)
+OutputWW = zeros(nvertelems*Nqk)
+OutputWRHO = zeros(nvertelems*Nqk)
+for ev in 1:nvertelems
+	for k in 1:Nqk
+		i=k + Nqk * (ev - 1)
+		OutputHF[i] = S_avg[k,ev,1]
+		OutputWQVAP[i] = S_avg[k,ev,2]
+		OutputWU[i] = S_avg[k,ev,3]
+		OutputWV[i] = S_avg[k,ev,4]
+		OutputWW[i] = S_avg[k,ev,5]
+		OutputWRHO[i] = S_avg[k,ev,6]
+	end
+end
+open("/home/yassine/yt-2T-drive/HF.txt", "w") do f
+writedlm("/home/yassine/yt-2T-drive/HF.txt", OutputHF)
+end
+open("/home/yassine/yt-2T-drive/WQVAP.txt", "w") do f
+writedlm("/home/yassine/yt-2T-drive/WQVAP.txt", OutputWQVAP)
+end
+open("/home/yassine/yt-2T-drive/WU.txt", "w") do f
+writedlm("/home/yassine/yt-2T-drive/WU.txt", OutputWU)
+end
+open("/home/yassine/yt-2T-drive/WV.txt", "w") do f
+writedlm("/home/yassine/yt-2T-drive/WV.txt", OutputWV)
+end
+open("/home/yassine/yt-2T-drive/WW.txt", "w") do f
+writedlm("/home/yassine/yt-2T-drive/WW.txt", OutputWW)
+end
+open("/home/yassine/yt-2T-drive/WRHO.txt", "w") do f
+writedlm("/home/yassine/yt-2T-drive/WRHO.txt", OutputWRHO)
```

Comment:
Better to write a new file whose name should contain the current step every time that the stats are calculated rather than overwriting them. We may want to track the evolution of the statistics as time progresses

Reply:
Can we append to the same file? Or even better yet, use some csv or xml format on the files so it is easy to plot the time evolution of the diagnostics.

I've dealt with grabbing and plotting numerical values from many texts files before and it's better that we avoiding building the machinery to do that. Much better to use a standard csv or xml import from a single file.

Reply:
Any objections to `Dataframe`s?